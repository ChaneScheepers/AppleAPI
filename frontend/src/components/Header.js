import React from "react";
const Header = () => {
  return (
      <div className="header-container">
          iTunes API search App
      </div>
  );
};

export default Header;
