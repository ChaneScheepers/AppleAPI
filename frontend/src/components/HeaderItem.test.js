import renderer from "react-test-renderer";
import Header from "././Header";


//snapshot test for the header comp.
test("FavItem basic render", () => {
  const component = renderer.create(
  <Header></Header>,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
