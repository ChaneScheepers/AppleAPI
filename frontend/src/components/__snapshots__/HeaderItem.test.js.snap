// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavItem basic render 1`] = `
<div
  className="header-container"
>
  iTunes API search App
</div>
`;
